sdk/include/opentelemetry/sdk/metrics/aggregator/aggregator.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include "opentelemetry/version.h"
+#include "opentelemetry/metrics/instrument.h"
+
+#include <variant>
+#include <vector>
+#include <mutex>
```

Comment:
nit - is there a reason these are not alphabetically sorted? If not, I would suggest to sort them.

Reply:
the format tools seem to reorder these